Phonebook/local-values.yml

```diff
@@ -19,14 +19,33 @@ source:
 
 traefik:
   enabled: true
-  startupArguments:
-    - ''
+  additional:
+    checkNewVersion: false
+    sendAnonymousUsage: false
   dashboard:
-    enabled: true
-    domain: board.demo-phonebook.local
-    ingress:
-      annotations:
-        traefik.frontend.priority: '100'
+    # Enable the dashboard on Traefik
+    enable: true
+    # Expose the dashboard and api through an ingress route at /dashboard
+    # and /api This is not secure and SHOULD NOT be enabled on production
+    # deployments
+    ingressRoute: true
+
+  additionalArguments:
+    - '--providers.kubernetesingress'
+    - '--api.insecure=true'
```

Comment:
Question: `api.insecure` that the APIs behind the traefik proxy can be http or https with an invalid cert?

Reply:
thats only for local development and is there in order to show the traefik dashboard.